net/libfab/libfab.h

```diff
@@ -0,0 +1,101 @@
+/* -*- C -*- */
+/*
+ * Copyright (c) 2019-2020 Seagate Technology LLC and/or its Affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com or cortx-questions@seagate.com.
+ *
+ */
+
+
+#pragma once
+
+#ifndef __MOTR_NET_LIBFAB_H__
+#define __MOTR_NET_LIBFAB_H__
+
+#include "rdma/fi_eq.h"
+#include "rdma/fi_domain.h"
+#include "rdma/fi_endpoint.h"
+#include "rdma/fabric.h"
+
+/**
+ * @defgroup netlibfab
+ *
+ * @{
+ */
+
+// #define FI_MAJOR_VERSION 1
+// #define FI_MINOR_VERSION 1
+
+struct libfabric_domain_params {
+	struct fi_info    *fi;
+	struct fi_info    *hints;
+	struct fid_fabric *fabric;
+	struct fid_domain *domain;
+};
+struct libfab_params {
+	struct fid_mr     *mr;
```

Comment:
* One member per line.
* align these members.
* add Doxygen comments for each member.

Reply:
most of the members are commented, some of them will be added in upcoming tasks.
We will be removing unwanted members after completion of all upcoming tasks.